.gitignore

```diff
@@ -2,6 +2,8 @@
 coverage/*
 logs/*
 node_modules/*
+package-lock.json
+package.json
```

Comment:
I would suggest not do this. In case a new package needs to be added, then it'll update the package and package lock. In general, this is not recommended. We can check this on a PR level. Please revert this.

Reply:
Done